creating-statistics/ud.qmd

```diff
@@ -608,6 +608,8 @@ The geographic_level column should describe the level of data present in that ro
 
 Teams should make sure that they are regularly checking their geography codes if they are not using a lookup from a maintained database (such as in the PDR). ONS have the [Open Geography portal](http://geoportal.statistics.gov.uk/){target="_blank" rel="noopener noreferrer"}, which can be a useful way of checking these. There is a wealth of data on there, though Local authority boundaries can be hard to find, they can be found using the tabs at the top – Boundaries > Administrative Boundaries > Counties and Unitary Authorities.
 
+Teams can also use the lookup tables available via the [data screener GitHub repository data folder](https://github.com/dfe-analytical-services/dfe-published-data-qa/tree/master/data). Guidance on how to link directly to the latest data in your code can be seen in the [standardised reference data section](rap.html#Standardised_reference_data)
```

Comment:
Make data screener link to github open in new tab, internal link to RAP is fine as is